js/ui/iconGrid.js

```diff
@@ -833,9 +833,9 @@ const IconGrid = new Lang.Class({
             dndDropPosition[1] -= Math.floor(oldIcon.get_size()[1] / 2);
 
             // Draw it at the location where DnD accept occured
-            let releaseOffset = this._findRelativeOffset(oldIcon, dndDropPosition);
-            oldIcon.translation_x = releaseOffset[0];
-            oldIcon.translation_y = releaseOffset[1];
+            //let releaseOffset = this._findRelativeOffset(oldIcon, dndDropPosition);
+            //oldIcon.translation_x = releaseOffset[0];
+            //oldIcon.translation_y = releaseOffset[1];
```

Comment:
Please remove the code, instead of commenting out.

Also, if this is truly temporary, I think it would be nice to comment on the commit message **why** it's temporary and/or what needs to be done to get a "more permanent" fix.